src/commands/sentinel-info-cache.json

```diff
@@ -26,7 +26,7 @@
                         },
                         {
                             "type": "array",
-                            "description": "The unix timestamp at which the logged command was processed.",
+                            "description": "This is actually a map, the first is the instance's INFO cached milliseconds, and the second is the instance's cached INFO string.",
```

Comment:
Is it first and second or odd and even?

The array above is also a map

Reply:
perhaps we need to describe each of these two maps separately
anyway, here is the format and example, can you suggest a better text for me? (make the call...)

the reply format:
```
        /* Reply format:
         *   1) master name
         *   2) 1) 1) info cached ms
         *         2) info from master
         *      2) 1) info cached ms
         *         2) info from replica1
         *      ...
         *   3) other master name
         *      ...
         *   ...
         */
```

an example:
```
127.0.0.1:26379> sentinel info-cache mymaster
1) "mymaster"
2) 1) 1) (integer) 6838
      2) "# Server\r\nredis_version balabal... from master"
   2) 1) (integer) 479
      2) "# Server\r\nredis_version balabal... from replica"
3) "mymaster2"
4) 1) 1) (integer) 7870
      2) "# Server\r\nredis_version balabal... from master2"
   2) 1) (integer) 3316
      2) "# Server\r\nredis_version:balabal... from replica"
```

Reply:
i looked at the code and realized this inner array is actually not a map (since the age for multiple entries can be identical).
so the other one can be documented as a map, but the inner one cannot.
i made some changes, someone please ack.

Reply:
the changes LGTM. thanks for the cleanups